tutorials/workflow/python/combined-patterns/shipping_app/models.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass
+
+@dataclass
+class ShipmentRegistrationStatus:
+    order_id: str
+    is_success: bool
+    message: str = None
```

Comment:
```suggestion
    message: Optional[str] = None
```

This is a good practice, since you are allowing None as value